cime_config/testdefs/testmods_dirs/clm/cropAnnOutputMonthly/user_nl_clm

```diff
@@ -0,0 +1,3 @@
+! These variables SHOULD only be saved annually in real runs, but for testing purposes it's fine to have them monthly.
+! Modifies h2 history file defined in crop testmod.
+hist_nhtfrq(3) = 0
```

Comment:
Check to see if this testmod is really needed, now that h2 in cropMonthOutput is saved monthly.

Reply:
Nope, redundant with `cropMonthOutput`. Deleted in fecad2997.